src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsAutoListMapping.java

```diff
@@ -82,6 +83,8 @@ public static boolean doesNotHaveDwpResponseDate(@Valid SscsCaseData caseData) {
 
     public static boolean hasMqpmOrFqpm(@Valid SscsCaseData caseData, ReferenceDataServiceHolder referenceData) {
         SessionCategoryMap sessionCategoryMap = getSessionCaseCode(caseData, referenceData);
+        Objects.requireNonNull(sessionCategoryMap, "sessionCategoryMap is null. The benefit/issue code is probably an incorrect combination and cannot be mapped"
+            + " to a session code. Refer to the session-category-map.json file for the correct combinations.");
```

Comment:
Minor suggestion, seem like hearing API usually use defined exception, such as InvalidMappingException for exception throwing. would it be good consistent will others?
![image](https://user-images.githubusercontent.com/93123095/190590255-5fd9b7cb-3141-4f8c-9ea2-d83a70845989.png)